core/server/master/pom.xml

```diff
@@ -70,6 +70,10 @@
       <groupId>org.glassfish.jersey.core</groupId>
       <artifactId>jersey-server</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.inject</groupId>
```

Comment:
is this package used anywhere?

Reply:
http://blog.supol.cz/?p=45

Reply:
I think this link provides a better explanation: https://eclipse-ee4j.github.io/jersey.github.io/release-notes/2.26.html